test/DefaultCluster.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+﻿using Orleans.CodeGeneration;
+using UnitTests.FSharpTypes;
+
+[assembly: KnownAssembly(typeof(SingleCaseDU))]
```

Comment:
The reason I added the fix to InternalGrains project is that we've codegen enabled on that, so it will work with build time codegen. Here this will trigger Runtime codegen what is not what we want, but fixes the previous test issue.